docfx_yaml/extension.py

```diff
@@ -720,23 +720,42 @@ def sanitize_uidname_field(toc_yaml):
                 sanitize_uidname_field(module['items'])
             module.pop('uidname')
 
-    # Parses the package name and returns package name and module name.
-    def find_package_name(package_name):
+    # Parses the package name and returns unique identifer and name.
+    def find_unique_name(package_name, entries):
```

Comment:
I hate to say, this is begging for a unit test. It appears to be self contained and we know exactly what the output should be based on the given input?

Reply:
Could this wait until #44 ? I'm planning on addressing all of this with that.

Reply:
I don't think so. This wouldn't be a test for verifying output, it would be a test of this function.

Right now, reviewing this, it seems OK. But, a test would make me feel much better about it.

Reply:
Sounds good.